### ISSUE TITLE
No sound during calls on both ends 

### ISSUE THREAD

**Issue by gonviegas** [telegramdesktop/tdesktop#26070]
### Steps to reproduce

Make a call

### Expected behaviour

Have sound during call

### Actual behaviour

No sound during call on both ends

### Operating system

Debian 11.6 - dwm

### Version of Telegram Desktop

4.7

### Installation source

Static binary from official website

### Logs

```shell
[2023.03.21 11:56:26] Launched version: 4007000, install beta: [TRUE], alpha: 0, debug mode: [FALSE]
[2023.03.21 11:56:26] Executable dir: /home/gonv/.local/bin/, name: Telegram
[2023.03.21 11:56:26] Initial working dir: /home/gonv/
[2023.03.21 11:56:26] Working dir: /home/gonv/.local/share/TelegramDesktop/
[2023.03.21 11:56:26] Command line: Telegram
[2023.03.21 11:56:26] Executable path before check: /home/gonv/.local/bin/Telegram
[2023.03.21 11:56:26] Logs started
[2023.03.21 11:56:26] Launcher filename: org.telegram.desktop._22e43c3e3c431fee83b0cdd98b3c69f4.desktop
[2023.03.21 11:56:26] Connecting local socket to /tmp/0ed5a14f9211c4d57167535ab470c6db-{87A94AB0-E370-4cde-98D3-ACC110C5967D}...
[2023.03.21 11:56:26] This is the only instance of Telegram, starting server and app...
[2023.03.21 11:56:26] Moved logging from '/home/gonv/.local/share/TelegramDesktop/log_start0.txt' to '/home/gonv/.local/share/TelegramDesktop/log.txt'!
[2023.03.21 11:56:26] Opened '/home/gonv/.local/share/TelegramDesktop/tdata/working' for reading, the previous Telegram Desktop launch was not finished properly :( Crash log size: 0
[2023.03.21 11:56:26] Global devicePixelRatio: 1
[2023.03.21 11:56:26] Primary screen DPI: 96, Base: 96.
[2023.03.21 11:56:26] Computed screen scale: 100
[2023.03.21 11:56:26] DevicePixelRatio: 1
[2023.03.21 11:56:26] ScreenScale: 100
[2023.03.21 11:56:26] System tray available: [TRUE]
[2023.03.21 11:56:26] Font: from ':/gui/fonts/DAOpenSansRegular.ttf' loaded 'DAOpenSansRegular'
[2023.03.21 11:56:26] Font: from ':/gui/fonts/DAVazirRegular.ttf' loaded 'DAVazirRegular'
[2023.03.21 11:56:26] Font: from ':/gui/fonts/DAOpenSansRegularItalic.ttf' loaded 'DAOpenSansRegularItalic'
[2023.03.21 11:56:26] Font: from ':/gui/fonts/DAVazirRegular.ttf' loaded 'DAVazirRegular'
[2023.03.21 11:56:26] Font: from ':/gui/fonts/DAOpenSansSemiboldAsBold.ttf' loaded 'DAOpenSansSemibold'
[2023.03.21 11:56:26] Font: from ':/gui/fonts/DAVazirMediumAsBold.ttf' loaded 'DAVazirMedium'
[2023.03.21 11:56:26] Font: from ':/gui/fonts/DAOpenSansSemiboldItalicAsBold.ttf' loaded 'DAOpenSansSemiboldItalic'
[2023.03.21 11:56:26] Font: from ':/gui/fonts/DAVazirMediumAsBold.ttf' loaded 'DAVazirMedium'
[2023.03.21 11:56:26] Font: from ':/gui/fonts/DAOpenSansSemiboldAsBold.ttf' loaded 'DAOpenSansSemibold'
[2023.03.21 11:56:26] Font: from ':/gui/fonts/DAVazirMediumAsBold.ttf' loaded 'DAVazirMedium'
[2023.03.21 11:56:26] Font: from ':/gui/fonts/DAOpenSansSemiboldItalicAsBold.ttf' loaded 'DAOpenSansSemiboldItalic'
[2023.03.21 11:56:26] Font: from ':/gui/fonts/DAVazirMediumAsBold.ttf' loaded 'DAVazirMedium'
[2023.03.21 11:56:26] Icon theme: Papirus-Dark
[2023.03.21 11:56:26] Fallback icon theme:
[2023.03.21 11:56:26] App Info: reading settings...
[2023.03.21 11:56:26] App Info: reading encrypted settings...
[2023.03.21 11:56:26] Lang Info: Loaded cached, keys: 4771
[2023.03.21 11:56:26] OpenAL Logging Level: (not set)
[2023.03.21 11:56:26] Audio Playback Devices: Family 17h (Models 10h-1fh) HD Audio Controller Analog Stereo
[2023.03.21 11:56:26] Audio Playback Default Device: Family 17h (Models 10h-1fh) HD Audio Controller Analog Stereo
[2023.03.21 11:56:26] Audio Capture Devices: Trust GXT 232 Microphone Mono;Monitor of Family 17h (Models 10h-1fh) HD Audio Controller Analog Stereo
[2023.03.21 11:56:26] Audio Capture Default Device: Trust GXT 232 Microphone Mono
[2023.03.21 11:56:27] App Info: reading accounts info...
[2023.03.21 11:56:27] App Info: reading encrypted info...
[2023.03.21 11:56:27] App Info: reading map...
[2023.03.21 11:56:27] App Info: reading encrypted map...
[2023.03.21 11:56:27] App Info: reading encrypted user settings...
[2023.03.21 11:56:27] App Info: encrypted user settings read.
[2023.03.21 11:56:27] App Info: reading encrypted mtp data...
[2023.03.21 11:56:27] MTP Info: read keys, current: 4, to destroy: 0
[2023.03.21 11:56:27] Map read time: 2
[2023.03.21 11:56:27] App Info: reading encrypted mtp config...
[2023.03.21 11:56:27] Export Info: Destroy top bar by controller removal.
[2023.03.21 11:56:27] OpenGL Profile: Compatibility.
[2023.03.21 11:56:27] OpenGL Renderer: AMD RENOIR (DRM 3.48.0, 6.0.0-0.deb11.6-amd64, LLVM 11.0.1)
[2023.03.21 11:56:27] OpenGL Vendor: AMD
[2023.03.21 11:56:27] OpenGL Version: 4.6 (Compatibility Profile) Mesa 20.3.5
[2023.03.21 11:56:27] OpenGL Extensions: GL_ARB_vertex_type_10f_11f_11f_rev, GL_ARB_shader_draw_parameters, GL_ARB_base_instance, GL_ARB_draw_buffers, GL_ARB_polygon_offset_clamp, GL_ARB_indirect_parameters, GL_ARB_half_float_vertex, GL_ARB_explicit_attrib_location, GL_ARB_texture_query_lod, GL_AMD_texture_texture4, GL_EXT_texture_swizzle, GL_NV_copy_image, GL_ARB_texture_non_power_of_two, GL_ARB_timer_query, GL_ARB_pixel_buffer_object, GL_AMD_shader_stencil_export, GL_ARB_shader_atomic_counter_ops, GL_ARB_seamless_cubemap_per_texture, GL_AMD_performance_monitor, GL_ARB_texture_rg, GL_ARB_spirv_extensions, GL_ARB_texture_gather, GL_ARB_texture_mirrored_repeat, GL_ARB_shader_bit_encoding, GL_ARB_compressed_texture_pixel_storage, GL_ARB_shader_clock, GL_ARB_vertex_attrib_64bit, GL_EXT_blend_func_separate, GL_NV_half_float, GL_NV_texture_barrier, GL_NV_primitive_restart, GL_EXT_texture_integer, GL_ARB_texture_env_dot3, GL_ARB_stencil_texturing, GL_EXT_texture3D, GL_EXT_shader_image_load_formatted, GL_ARB_ES3_compatibility, GL_EXT_timer_query, GL_EXT_blend_color, GL_MESA_framebuffer_flip_y, GL_ARB_occlusion_query, GL_AMD_multi_draw_indirect, GL_ARB_conditional_render_inverted, GL_ARB_texture_buffer_range, GL_ARB_shader_texture_lod, GL_ARB_separate_shader_objects, GL_EXT_window_rectangles, GL_IBM_texture_mirrored_repeat, GL_ARB_blend_func_extended, GL_ARB_depth_clamp, GL_ARB_fragment_program_shadow, GL_ARB_bindless_texture, GL_ARB_multi_draw_indirect, GL_ARB_texture_storage_multisample, GL_APPLE_packed_pixels, GL_EXT_EGL_sync, GL_ARB_direct_state_access, GL_NV_alpha_to_coverage_dither_control, GL_SUN_multi_draw_arrays, GL_ARB_tessellation_shader, GL_KHR_parallel_shader_compile, GL_INGR_blend_func_separate, GL_ARB_shader_ballot, GL_ARB_vertex_type_2_10_10_10_rev, GL_NV_texture_env_combine4, GL_AMD_conservative_depth, GL_ARB_multisample, GL_ARB_texture_rectangle, GL_ARB_depth_buffer_float, GL_EXT_framebuffer_multisample, GL_ANGLE_texture_compression_dxt5, GL_EXT_secondary_color, GL_ARB_texture_filter_anisotropic, GL_ARB_texture_float, GL_ARB_ES2_compatibility, GL_SGIS_texture_edge_clamp, GL_ARB_uniform_buffer_object, GL_ARB_get_texture_sub_image, GL_ATI_separate_stencil, GL_AMD_query_buffer_object, GL_ARB_multi_bind, GL_ARB_occlusion_query2, GL_EXT_transform_feedback, GL_AMD_seamless_cubemap_per_texture, GL_ARB_texture_query_levels, GL_EXT_direct_state_access, GL_ATI_texture_mirror_once, GL_ARB_transform_feedback3, GL_ARB_texture_storage, GL_ARB_texture_border_clamp, GL_EXT_memory_object, GL_NV_blend_square, GL_EXT_draw_instanced, GL_EXT_shader_image_load_store, GL_EXT_provoking_vertex, GL_EXT_texture_rectangle, GL_ARB_vertex_array_object, GL_EXT_texture_array, GL_MESA_pack_invert, GL_EXT_polygon_offset_clamp, GL_ARB_draw_indirect, GL_EXT_copy_texture, GL_EXT_texture_filter_anisotropic, GL_ARB_texture_env_combine, GL_ARB_arrays_of_arrays, GL_ARB_seamless_cube_map, GL_AMD_framebuffer_multisample_advanced, GL_MESA_texture_signed_rgba, GL_EXT_draw_range_elements, GL_ARB_internalformat_query2, GL_EXT_texture_compression_dxt1, GL_ARB_shader_stencil_export, GL_ARB_texture_cube_map_array, GL_EXT_shadow_funcs, GL_ATI_texture_env_combine3, GL_MESA_window_pos, GL_ARB_gpu_shader_int64, GL_ARB_copy_buffer, GL_ARB_compute_variable_group_size, GL_ARB_gpu_shader5, GL_NV_shader_atomic_int64, GL_ARB_shader_image_load_store, GL_ARB_conservative_depth, GL_ARB_invalidate_subdata, GL_EXT_packed_float, GL_EXT_packed_pixels, GL_AMD_shader_trinary_minmax, GL_ARB_texture_env_add, GL_EXT_texture_snorm, GL_ARB_vertex_program, GL_EXT_shader_integer_mix, GL_KHR_context_flush_control, GL_NV_depth_clamp, GL_ARB_draw_buffers_blend, GL_EXT_vertex_array_bgra, GL_EXT_texture_shared_exponent, GL_ARB_transpose_matrix, GL_AMD_vertex_shader_layer, GL_ARB_texture_barrier, GL_KHR_blend_equation_advanced, GL_AMD_draw_buffers_blend, GL_ARB_compatibility, GL_ARB_color_buffer_float, GL_ARB_query_buffer_object, GL_EXT_EGL_image_storage, GL_ARB_clear_buffer_object, GL_ARB_sampler_objects, GL_ARB_provoking_vertex, GL_ARB_ES3_1_compatibility, GL_EXT_depth_bounds_test, GL_ATI_blend_equation_separate, GL_EXT_texture_edge_clamp, GL_ARB_internalformat_query, GL_ARB_derivative_control, GL_ARB_framebuffer_object, GL_ARB_sample_shading, GL_ARB_shader_precision, GL_IBM_multimode_draw_arrays, GL_ARB_robust_buffer_access_behavior, GL_ARB_parallel_shader_compile, GL_EXT_framebuffer_multisample_blit_scaled, GL_ARB_pipeline_statistics_query, GL_ARB_shader_texture_image_samples, GL_ARB_framebuffer_sRGB, GL_ARB_texture_compression, GL_EXT_packed_depth_stencil, GL_EXT_texture_sRGB_decode, GL_EXT_pixel_buffer_object, GL_ARB_compute_shader, GL_ARB_transform_feedback_overflow_query, GL_NV_texture_rectangle, GL_OES_EGL_image, GL_EXT_texture_env_dot3, GL_EXT_texture_cube_map, GL_EXT_compiled_vertex_array, GL_ARB_robustness, GL_KHR_robustness, GL_EXT_abgr, GL_NV_vdpau_interop, GL_ARB_texture_cube_map, GL_ARB_shader_image_size, GL_EXT_texture_object, GL_ARB_vertex_buffer_object, GL_EXT_point_parameters, GL_ARB_texture_mirror_clamp_to_edge, GL_NV_texgen_reflection, GL_EXT_subtexture, GL_ARB_texture_compression_bptc, GL_ARB_instanced_arrays, GL_ARB_texture_stencil8, GL_OES_read_format, GL_ARB_draw_elements_base_vertex, GL_EXT_blend_minmax, GL_IBM_rasterpos_clip, GL_EXT_texture_buffer_object, GL_ARB_clear_texture, GL_ARB_cull_distance, GL_ARB_depth_texture, GL_ARB_fragment_coord_conventions, GL_ANGLE_texture_compression_dxt3, GL_ARB_shader_group_vote, GL_INTEL_blackhole_render, GL_EXT_vertex_array, GL_ARB_point_parameters, GL_EXT_fog_coord, GL_SGIS_texture_border_clamp, GL_ARB_shader_subroutine, GL_ARB_fragment_program, GL_ARB_viewport_array, GL_ARB_enhanced_layouts, GL_EXT_texture_shadow_lod, GL_ATI_draw_buffers, GL_EXT_texture_compression_latc, GL_EXT_separate_specular_color, GL_SGIS_texture_lod, GL_ARB_shader_objects, GL_ARB_program_interface_query, GL_AMD_vertex_shader_viewport_index, GL_ARB_fragment_layer_viewport, GL_EXT_framebuffer_blit, GL_ARB_shadow, GL_EXT_blend_equation_separate, GL_EXT_framebuffer_object, GL_EXT_draw_buffers2, GL_EXT_framebuffer_sRGB, GL_ARB_gpu_shader_fp64, GL_ARB_texture_swizzle, GL_ARB_multitexture, GL_EXT_stencil_two_side, GL_ARB_texture_view, GL_ARB_texture_multisample, GL_EXT_gpu_program_parameters, GL_ARB_copy_image, GL_EXT_texture_env_add, GL_ARB_transform_feedback2, GL_EXT_memory_object_fd, GL_MESA_shader_integer_functions, GL_ARB_texture_buffer_object_rgb32, GL_ATI_texture_float, GL_EXT_semaphore, GL_NVX_gpu_memory_info, GL_EXT_texture_lod_bias, GL_EXT_texture_sRGB_R8, GL_ATI_meminfo, GL_ARB_explicit_uniform_location, GL_EXT_shader_samples_identical, GL_ARB_half_float_pixel, GL_NV_copy_depth_to_color, GL_EXT_texture_sRGB, GL_ARB_shading_language_packing, GL_ARB_texture_compression_rgtc, GL_EXT_gpu_shader4, GL_ARB_draw_instanced, GL_KHR_texture_compression_astc_sliced_3d, GL_ARB_map_buffer_range, GL_ATI_texture_compression_3dc, GL_ARB_shader_viewport_layer_array, GL_KHR_no_error, GL_KHR_robust_buffer_access_behavior, GL_ARB_vertex_array_bgra, GL_ARB_point_sprite, GL_EXT_texture_mirror_clamp, GL_ARB_fragment_shader, GL_EXT_texture_env_combine, GL_EXT_stencil_wrap, GL_AMD_pinned_memory, GL_EXT_texture_compression_rgtc, GL_KHR_texture_compression_astc_ldr, GL_ARB_vertex_attrib_binding, GL_ARB_map_buffer_alignment, GL_ATI_fragment_shader, GL_EXT_semaphore_fd, GL_ARB_get_program_binary, GL_SGIS_generate_mipmap, GL_ARB_shader_atomic_counters, GL_ARB_texture_rgb10_a2ui, GL_ARB_shading_language_100, GL_ARB_sparse_buffer, GL_ARB_debug_output, GL_EXT_bgra, GL_NV_packed_depth_stencil, GL_ARB_clip_control, GL_AMD_depth_clamp_separate, GL_ARB_shader_storage_buffer_object, GL_KHR_debug, GL_EXT_texture, GL_ARB_buffer_storage, GL_NV_conditional_render, GL_ARB_gl_spirv, GL_ARB_texture_buffer_object, GL_ARB_window_pos, GL_ARB_shading_language_include, GL_NV_fog_distance, GL_EXT_vertex_attrib_64bit, GL_ARB_sync, GL_EXT_multi_draw_arrays, GL_ARB_framebuffer_no_attachments, GL_ARB_texture_env_crossbar, GL_ARB_shading_language_420pack, GL_ARB_vertex_shader, GL_NV_light_max_exponent, GL_EXT_blend_subtract, GL_EXT_rescale_normal, GL_S3_s3tc, GL_EXT_texture_compression_s3tc, GL_ARB_transform_feedback_instanced
[2023.03.21 11:56:27] OpenGL: QOpenGLContext created, version: 4.4.
[2023.03.21 11:56:27] OpenGL: [TRUE] (Window)
[2023.03.21 11:56:28] Message Info: bad message notification received (error_code 33) for msg_id = 7212967163561178872, seq_no = 50
[2023.03.21 11:56:28] Message Info: bad message notification received, msgId 7212967163561178872, error_code 33
[2023.03.21 12:04:49] OpenGL: [TRUE] (Window)
[2023.03.21 12:05:03] Audio Info: recreating audio device and reattaching the tracks
[2023.03.21 12:05:06] Call Info: Creating instance with version '8.0.0', allowP2P: [TRUE]
[2023.03.21 12:05:07] Call Error: Unexpected state in handleStateChange: 4
[2023.03.21 12:05:07] Call Error: Unexpected state in handleStateChange: 4
[2023.03.21 12:05:08] Audio Info: Closing audio playback device.
[2023.03.21 12:05:26] Audio Info: recreating audio device and reattaching the tracks
[2023.03.21 12:05:28] Audio Info: Closing audio playback device.
[2023.03.21 12:05:35] Audio Info: recreating audio device and reattaching the tracks
[2023.03.21 12:05:37] Audio Info: Closing audio playback device.
[2023.03.21 12:05:45] Audio Info: recreating audio device and reattaching the tracks
[2023.03.21 12:05:47] Audio Info: Closing audio playback device.
[2023.03.21 12:06:23] Audio Info: recreating audio device and reattaching the tracks
[2023.03.21 12:06:25] Audio Info: Closing audio playback device.
```

**Comment by ilya-fedin** [comment]
Please provide the terminal output when running terminal with `ALSOFT_LOGLEVEL=3` environment variable set

**Comment by Aokromes** [comment]
also try on 4.7.1

**Comment by gonviegas** [comment]
> Please provide the terminal output when running terminal with `ALSOFT_LOGLEVEL=3` environment variable set

```
./Telegram
[W][28813.685184] pw.conf      | [          conf.c:  939 try_load_conf()] can't load config client-rt.conf: No such file or directory
[W][28813.685258] pw.conf      | [          conf.c:  939 try_load_conf()] can't load config client.conf: No such file or directory
[E][28813.685276] pw.conf      | [          conf.c:  963 pw_conf_load_conf_for_context()] can't load default config client.conf: No such file or directory
[ALSOFT] (EE) Failed to create PipeWire event context (errno: 2)
QPainter::begin: Paint device returned engine == 0, type: 2
QWidget::render: Cannot render with an inactive painter
qt.qpa.xcb: Unhandled client message:  "_NET_WM_NAME"
qt.qpa.xcb: Unhandled client message:  "_NET_WM_NAME"
qt.qpa.xcb: Unhandled client message:  "_NET_WM_NAME"
qt.qpa.xcb: Unhandled client message:  "_NET_WM_NAME"
qt.gui.imageio.jpeg: Corrupt JPEG data: premature end of data segment
qt.gui.imageio.jpeg: Corrupt JPEG data: premature end of data segment
D/tgvoip: === Updating voip config ===
D/tgvoip: {"enable_vp8_encoder":true,"enable_vp8_decoder":true,"enable_vp9_encoder":true,"enable_vp9_decoder":true,"enable_h265_encoder":true,"enable_h265_decoder":true,"enable_h264_encoder":true,"enable_h264_decoder":true,"audio_frame_size":60,"jitter_min_delay_60":2,"jitter_max_delay_60":10,"jitter_max_slots_60":20,"jitter_losses_to_reset":20,"jitter_resync_threshold":0.5,"audio_congestion_window":1024,"audio_max_bitrate":20000,"audio_max_bitrate_edge":16000,"audio_max_bitrate_gprs":8000,"audio_max_bitrate_saving":8000,"audio_init_bitrate":16000,"audio_init_bitrate_edge":8000,"audio_init_bitrate_gprs":8000,"audio_init_bitrate_saving":8000,"audio_bitrate_step_incr":1000,"audio_bitrate_step_decr":1000,"use_system_ns":true,"use_system_aec":true,"force_tcp":false,"jitter_initial_delay_60":2,"adsp_good_impls":"(Qualcomm Fluence)","bad_call_rating":true,"use_ios_vpio_agc":false,"use_tcp":false,"audio_medium_fec_bitrate":20000,"audio_medium_fec_multiplier":0.1,"audio_strong_fec_bitrate":7000}
```

**Comment by ilya-fedin** [comment]
@gonviegas the output doesn't look like you have the variable set

**Comment by gonviegas** [comment]
> @gonviegas the output doesn't look like you have the variable set

Working now on latest released version 4.7.1.
